lib/sober_swag/input_object.rb

```diff
@@ -12,24 +12,67 @@ class InputObject < Dry::Struct
     class << self
       ##
       # The name to use for this type in external documentation.
-      def identifier(arg = nil)
-        @identifier = arg if arg
+      #
+      # @param new_ident [String] what to call this InputObject in external documentation.
+      def identifier(new_ident = nil)
+        @identifier = new_ident if new_ident
 
         @identifier || name.to_s.gsub('::', '.')
       end
 
+      ##
+      # @overload attribute(key, parent = SoberSwag::InputObject, &block)
+      #   Defines an attribute as a direct sub-object.
+      #   This block will be called as in {SoberSwag.input_object}.
+      #   This might be useful in a case like the following:
+      #
+      #   ```ruby
+      #   class Classroom < SoberSwag::InputObject do
```

Comment:
```suggestion
      #   class Classroom < SoberSwag::InputObject
```

Before:
![image](https://user-images.githubusercontent.com/3358255/108949204-392a3d80-7621-11eb-9e33-b13f39ad15fd.png)

After
![image](https://user-images.githubusercontent.com/3358255/108949182-2d3e7b80-7621-11eb-9395-0cb9f9db1606.png)